### PR TITLE
Polish Parent inbox and dashboard

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-13941
affects: Parent
release note: none

fixes:
- webview bg
- floating button on grades
- changed inbox to white navbar

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021243&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent

[MBL-13941](https://instructure.atlassian.net/browse/MBL-13941)

<details><summary>:x: XCTest failed: CoreTests/UINavigationBarExtensionsTests/testUseModalStyle</summary>


    Assertion Failure: UINavigationBarExtensionsTests.swift:72: XCTAssertFalse failed
</details>
<details><summary>:x: XCTest failed: ParentUnitTests/ConversationListViewControllerTests/testLayout</summary>


    Assertion Failure: ConversationListViewControllerTests.swift:46: XCTAssertFalse failed
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021247&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

File: Core/Core/Syllabus/SyllabusViewController.swift [hunk 1]
```diff
@@ -32,6 +32,7 @@ public class SyllabusViewController: UIViewController {
     public override func viewDidLoad() {
         super.viewDidLoad()
         webView.linkDelegate = self
+        webView.backgroundColor = .white
```
Comment (vanwagonet) [inline Core/Core/Syllabus/SyllabusViewController.swift]: in dark mode should it have a dark background? `.named(.backgroundLightest)` is how we've been handling these.

Comment (bluwave) [reply Core/Core/Syllabus/SyllabusViewController.swift]: I can change that.

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021272&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021273&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021275&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021278&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021279&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021393&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021405&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1021416&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|17
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by bluwave** [comment]
@vanwagonet - changed this back to a hidden nav bar approach, the wrong color status bar is still broken, won't fix in this pr

**Pull request merged**